test/router_test.exs

```diff
@@ -197,4 +197,56 @@ defmodule Kitto.RouterTest do
     assert conn.status == 401
     Application.delete_env :kitto, :auth_token
   end
+
+  test "POST dashboards/:id reloads single dashboard" do
+    dashboard = "sample"
+    body = %{event: "reload"}
```

Comment:
I think we should use `command` here instead of `event`.
The internal events namespace `_kitto` will be used for the backend to broadcast commands.
What do you think?